openedx/core/djangoapps/video_pipeline/config/waffle.py

```diff
@@ -8,10 +8,30 @@
 # Videos Namespace
 WAFFLE_NAMESPACE = 'videos'
 
-# Waffle flag telling whether youtube is deprecated.
+# .. toggle_name: videos.deprecate_youtube
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag telling whether youtube is deprecated. When
+#   enabled, videos are no longer uploaded to YouTube as part of the video pipeline.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2018-08-03
+# .. toggle_target_removal_date: None
+# .. toggle_warnings: None
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/18765
 DEPRECATE_YOUTUBE = 'deprecate_youtube'
+# .. toggle_name: videos.enable_devstack_video_uploads
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, use Multi-Factor Authentication (MFA) for
+#   authenticating to AWS. These short-lived access tokens are well suited for
+#   development (probably?).
+# .. toggle_use_cases: open_edx
```

Comment:
If you are only partially confident on any values, like the use case, please note that in the warning or description.

Reply:
:ok_hand: I added a few comments in the warning sections.